packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -17,88 +17,121 @@
 import {types} from '@opencensus/opencensus-core';
 import {classes} from '@opencensus/opencensus-core';
 import {logger} from '@opencensus/opencensus-core';
+import * as http from 'http';
 import * as semver from 'semver';
 import * as shimmer from 'shimmer';
 import * as url from 'url';
 import * as uuid from 'uuid';
 
-// import * as Debug from 'debug';
-// const debug = Debug('opencensus');
 
+// TODO: maybe we should have a setup as a Client or as Server.
+
+type HttpModuleType = typeof http;
+type RequestFunction = typeof http.request;
+
+/** Http instrumentation plugin for Opencensus */
 export class HttpPlugin extends classes.BasePlugin {
+  /**
+   * Atributes Names according to Opencensus HTTP Specs
+   * https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/HTTP.md
+   */
+  readonly ATRIBUTE_HTTP_HOST = 'http.host';
```

Comment:
Spelling: `ATTRIBUTE` (this should be changed in the docs as well)

Reply:
Also, these should be `static`. They're not properties of the plugin instance.

Reply:
Done!